add_nyt_review_link.py

```diff
@@ -69,6 +69,10 @@ def __add_link(self, work, link_struct) -> None:
                     .format(work.olid))
         except AttributeError:
             work.links = [link_struct]
+            self.logger.info(
```

Comment:
I don't think we need this log as it is logging expected behavior

Reply:
Yeah, but as we don't have comments calling the save function now, maybe we could try to compensate via logging in this place? I have no strong opinion about that though, so I would remove that info and probably another one that logs happy path several lines above.